housekeeper/store/filters/file_filters.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+
+from housekeeper.store.models import File
+
+
+def filter_files_by_id(files: Query, file_id: int, **kwargs) -> Query:
+    """Filter files by file id."""
+    return files.filter(File.id == file_id)
+
+
+class FileFilters(Enum):
+    """Define File filter functions."""
+    FILTER_BY_ID: Callable = filter_files_by_id
+
+
+def apply_file_filters(
+    files: Query,
+    filter_functions: List[Callable],
+    file_id: Optional[int] = None,
+) -> Query:
+    """Apply filtering functions and return filtered results."""
```

Comment:
Proposal. Results sound quite vague to me
```suggestion
    """Apply filtering functions and return filtered query."""
```